src/el/ethereumjs/ethereumjs_launcher.star

```diff
@@ -100,6 +103,10 @@ def launch(
 
 	jwt_secret = shared_utils.read_file_from_service(plan, service_name, jwt_secret_json_filepath_on_client)
 
+	# TODO: Passing empty string for metrics_url for now
+	# metrics_url = "http://{0}:{1}".format(service.ip_address, METRICS_PORT_NUM)
+	ethjs_metrics_info = node_metrics.new_node_metrics_info(service_name, METRICS_PATH, "")
```

Comment:
lets add an issue to the package tracking and put a comment here to it!

Reply:
Created #127 for this. Will add it to the comment.